### ISSUE TITLE
Can't download Snaps

### ISSUE THREAD

**Issue by TheDandyBerry** [rodit/SnapMod#257]
I can't seem to download Snaps using "Save to camera roll" or triple dot "Save" option. First shows "Save failed" error and 2nd just doesn't save the pic on my phone. Anyone managed to resolve this issue? I'm using the recommended Snapchat version and SnapMod 1.8.5

**Comment by RevealedSoulEven** [comment]
Yes that's a bug. If you turn on "Allow download stories" then you can't save snaps in chat from the 3-dot menu, it automatically downloads the snap to your gallery. Also the downloaded snap may not be visible in phone's gallery. Check this location for the file
/internal storage/Download/Snapmod

**Comment by TheDandyBerry** [comment]
Doesn't seem to work either way. With "Allow download stories" off and auto-save it doesn't save it at all. With the option on I see the "Save" button but it also does not work at all. I have Android Download Manager on if that matters.

**Comment by rodit** [comment]
logs

**Comment by TheDandyBerry** [comment]
Attaching logs
[LSPosed_2022-09-24T17_00_27.828232.zip](https://github.com/rodit/SnapMod/files/9638971/LSPosed_2022-09-24T17_00_27.828232.zip)

**Comment by TheDandyBerry** [comment]
No idea why but since today 3dot menu works and I can save the snaps this way.
